crates/outline/src/outline.rs

```diff
@@ -345,6 +364,25 @@ impl PickerDelegate for OutlineViewDelegate {
     }
 }
 
+fn get_outline_item_depth(
+    item: &OutlineItem<Anchor>,
+    range: &Range<usize>,
+    cursor_offset: &usize,
+) -> Option<usize> {
+    if range.contains(cursor_offset) {
+        Some(item.depth)
+    } else {
+        None
+    }
+}
+
+fn get_distance_to_closest_endpoint(range: &Range<usize>, cursor_offset: &usize) -> isize {
```

Comment:
```suggestion
fn get_distance_to_closest_endpoint(range: &Range<usize>, cursor_offset: usize) -> isize {
```

I think, `Copy` primitives are preferred to be passed by value in Rust, so let's do that.
Same applies above.